ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/wan_path_groups.schema.yml

```diff
@@ -21,14 +19,20 @@ keys:
           type: str
           description: Path-group name.
         id:
-          type: str
-          description: Path-group id.
+          type: int
+          convert_types:
+            - str
+          description: |-
+            Path-group id.
+
+            TODO: Required until an auto ID algorithm is implemented.
+          required: true
         description:
           type: str
           description: Additional information about the path-group for documentation purposes.
         ipsec:
           type: bool
-          description: Flag to configure IPsec on the carrier (default is True).
+          description: Flag to configure IPsec on the path_group (default is True).
```

Comment:
If the default is true, you should set that in the schema instead of writing it :)

Reply:
[703afd1](https://github.com/aristanetworks/ansible-avd/pull/3440/commits/703afd1054dca9d5d00eb04a41b1059f4f810265)